scripts/test_translator.py

```diff
@@ -75,10 +75,11 @@ def __init__(self, log_level: str, path: str, flags: Set[str] = None) -> None:
         self.reorganize_definitions = "reorganize_definitions" in flags
         self.emit_build_files = "emit_build_files" in flags
 
-    def translate(self, cc_db, ld_lib_path, extra_args: List[str] = []) -> RustFile:
+    def translate(self, cc_db: str, ld_lib_path: str, extra_args: List[str] = []) -> RustFile:
```

Comment:
Default argument can't be mutable, like `[]`.

Reply:
Again, used immutably.